jsonnet/telemeter/rules.libsonnet

```diff
@@ -147,6 +147,23 @@
                 max by(_id) (count_over_time(cluster:usage:workload:capacity_physical_cpu_cores:max:5m[1h:5m])) / scalar(count_over_time(vector(1)[1h:5m]))
               |||,
             },
+            {
+              // OpenShift Cluster vCPU-hours for the last hour.
+              // The divisor of scalar(count_over_time(vector(1)[1h:5m])) allows us to get an effective average value having "absent data" treated as 0.
+              // max(...) by (_id) is used ensure a single datapoint per cluster ID.
+              record: 'cluster:usage:workload:capacity_virtual_cpu_hours',
+              expr: |||
+                 max by(_id) (sum(sum_over_time(cluster:capacity_cpu_cores:sum{label_node_role_kubernetes_io = ''}[1h:5m]) / scalar(count_over_time(vector(1)[1h:5m]))))
+              |||,
+            },
+            {
+              // OpenShift Cluster virtual Instance-hours for the last hour.
+              // The divisor of scalar(count_over_time(vector(1)[1h:5m])) allows us to get an effective average value having "absent data" treated as 0.
+              record: 'cluster:usage:workload:capacity_virtual_instance_hours',
+              expr: |||
+                max by(_id) (count_over_time(cluster:capacity_cpu_cores:sum{label_node_role_kubernetes_io = ''}[1h:5m])) / scalar(count_over_time(vector(1)[1h:5m]))
```

Comment:
This metric appears to always be identical to `cluster:usage:workload:capacity_physical_instance_hours`, which always seems to be 1 (if the cluster has been around for a while), or a graph slowly scaling up to 1 (if it's a new cluster). What's the significance of this rule?

Reply:
Removed.